src/Components/ToggleSave/ToggleSave.js

```diff
@@ -4,7 +4,7 @@ import toggleOffImage from "../images/color toggle (1).png";
 import "../ToggleSave/ToggleSave.css";
 
 const ToggleSave = () => {
-  const [isToggled, setIsToggled] = useState(false);
+  const [isToggled, setIsToggled] = useState(true); // Start with the toggle off
 
   const handleToggle = () => {
     setIsToggled(!isToggled);
```

Comment:
I think we'd have to change this to setIsToggled(isToggled) if we change toggled to saved... Either way is fine though. It just confuses me to have toggled be not saved...